internal/pkg/cli/svc_deploy.go

```diff
@@ -336,6 +340,16 @@ func workloadManifest(in *workloadManifestInput) (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("apply environment %s override: %s", in.envName, err)
 	}
+	serviceInRegion, err := mft.IsServiceAvailableInRegion(in.targetEnv.Region)
+	if err != nil {
+		return nil, fmt.Errorf("%s available for region %s: %w ", in.svcType, in.targetEnv.Region, err)
```

Comment:
nit:
```suggestion
		return nil, fmt.Errorf("check %s availability for region %s: %w ", in.svcType, in.targetEnv.Region, err)
```

Reply:
Haha, I was commenting on this at the same time as @Lou1415926. Her suggestion is better!